project_start/express-api/routes/post.js

```diff
@@ -364,12 +371,14 @@ router.post("/:postId/new-comment", async (req, res, next) => {
     const { songId, userObjectId, comment } = req.body;
     const Comments = Parse.Object.extend("Comments");
     const currComment = new Comments();
+    var commentSentiment = sentiment.analyze(comment);
```

Comment:
Good work :)

Reply:
Just a suggestion, you could have divided this diff into two parts : one for adding sentiment analysis package and the other one for consumption (js files). This would have helped filter out parts that dont require review (like sentiment analysis library in this case) and the ones that actually require review.